frontend/src/helpers/http/parse-query-string/parse-query-string.helper.ts

```diff
@@ -0,0 +1,5 @@
+import { parse } from 'query-string';
+
+const parseQueryString = (query: string): Record<string, unknown> => parse(query, { arrayFormat: 'bracket' });
```

Comment:
```suggestion
const parseQueryString = <T =  Record<string, unknown>>(query: string): T => parse(query, { arrayFormat: 'bracket' });
```

Reply:
In this case I have error: Type 'ParsedQuery<string>' is not assignable to type 'T'.
  'T' could be instantiated with an arbitrary type which could be unrelated to 'ParsedQuery<string>'.ts(2322)